src/pages/index.js

```diff
@@ -38,7 +20,7 @@ export default function Home() {
   useEffect(() => {
```

Comment:
useEffect's callback function expects to return yet another function. Currently, you are returning a state. You may delete the line entirely (the return, I mean).